OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -210,6 +210,13 @@ public override bool Tick(Actor self)
 
 			var nextCell = path[path.Count - 1];
 
+			// Something else might have moved us, so the path is no longer valid.
+			if (!Util.AdjacentCells(self.World, Target.FromCell(self.World, mobile.ToCell)).Contains(nextCell))
+			{
+				path = EvalPath(BlockedByActor.Immovable);
```

Comment:
The `BlockedByActor.Immovable` argument depends on #16408. Will this behave correctly on the prep branch if called without it?

Reply:
When called on the prep branch without the argument the result is that the path will also try to avoid the current positions of all moving actors. This is not quite optimal, but not a deal-breaker at all since this is the standard behaviour for all path searches pre-#16408.